src/ray/common/ray_config_def.h

```diff
@@ -35,7 +35,7 @@ RAY_CONFIG(int64_t, ray_cookie, 0x5241590000000000)
 RAY_CONFIG(int64_t, handler_warning_timeout_ms, 1000)
 
 /// The duration between heartbeats sent by the raylets.
-RAY_CONFIG(int64_t, raylet_heartbeat_timeout_milliseconds, 100)
+RAY_CONFIG(int64_t, raylet_heartbeat_period_milliseconds, 100)
```

Comment:
Not a big deal, but I think "interval" would be more accurate.
Because "interval" means the time between 2 heartbeats is 100ms. While "period" sounds like one heartbeat would last for 100ms.

Reply:
Seems that we are already using "period" everywhere. Never mind.